pkg/internal/httpclient/helper.go

```diff
@@ -98,6 +98,9 @@ func setHeaders(req *http.Request, cfg *config.Config, requestData RequestData)
 	if cfg.PlatformID != "" {
 		req.Header.Set("X-Platform-Id", cfg.PlatformID)
 	}
+	if cfg.ExpandNodes != "" {
+		req.Header.Set("X-Expand-Response-Nodes", cfg.ExpandNodes)
```

Comment:
[req] O nome "correto" do header é `X-Expand-Responde-Nodes`, com a letra D na palavra Response
```suggestion
		req.Header.Set("X-Expand-Responde-Nodes", cfg.ExpandNodes)
```

Reply:
eu também estranhei isso mas na doc tá realmente responde no header. Na sdk de node também tá setado como X-Expand-Responde-Nodes 
<img width="787" alt="Captura de Tela 2025-05-26 às 16 10 26" src="https://github.com/user-attachments/assets/1fd20778-dfb8-4dca-93b8-e08009618e97" />